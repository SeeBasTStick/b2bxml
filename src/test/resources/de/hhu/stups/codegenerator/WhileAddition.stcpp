int main() {
    While while1;
    cout << while1.Inc(BInteger(1));
    return 0;
}