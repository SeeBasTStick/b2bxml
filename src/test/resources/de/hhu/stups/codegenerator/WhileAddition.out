public static void main(String[] args) {
    While while = new While();
    System.out.println(while.Inc(new BInteger(1)));
}